src/vivarium/interface/cli.py

```diff
@@ -75,22 +77,22 @@ def run(model_specification, results_directory, verbose, log, with_debugger):
     is provided, a subdirectory will be created with the same name as the
     MODEL_SPECIFICATION if one does not exist. Results will be written to a
     further subdirectory named after the start time of the simulation run."""
+    start = time()
```

Comment:
Pushed the results writing behavior out here until we get to our results feature.